sklearn/model_selection/_validation.py

```diff
@@ -180,6 +183,14 @@ def cross_validate(estimator, X, y=None, groups=None, scoring=None, cv=None,
         Make a scorer from a performance metric or loss function.
 
     """
+    if return_train_score == "warn":
+        warnings.warn("Computing training scores may affect "
+                      "performance significantly. This is the reason "
+                      "return_train_score will change its default value "
+                      "from True (current behaviour) to False in 0.22. "
+                      "Please set explicitly return_train_score to get "
+                      "rid of this warning.", FutureWarning)
```

Comment:
"may affect performance" is ambiguous: it could be interpreted as affecting the predictive accuracy. I would suggest: "can slow do the grid search significantly" or "may be computationally expensive".